### DISCUSSION TITLE
2023-08-20-edge

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#148] [General]
# 2023 我的浏览器折腾之旅 - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Share浏览器应该是不少同学电脑上使用频次最高的应用，对强迫症而言，会经常将干活的工具折腾到好用的状态。去年有写过我的 Mac 软件折腾之旅，经过一年的迭代变化，打算今年继续写，刚好有朋友问我用的是什么浏览器，那就从「2023 我的浏览器折腾之旅」开始吧。

https://tw93.fun/2023-08-20/edge.html

**Comment by LinxHex** [comment]
哈哈，我从Windows11回到Windows10LTSC的主要动力就是卸载Edge

**Reply by tw93** [reply]
哈哈哈不过有一说一 微软技术可以 但是产品设计真一般

**Reply by LinxHex** [reply]
经常猝不及防弹窗让我该设置，稍有不慎就上当，太可恶了。

**Reply by tw93** [reply]
Mac 上面其实会克制很多

**Reply by LinxHex** [reply]
反馈一下贵Blog建议，文章内有超链接的文字不太明显，建议优化下

**Comment by Tom89757** [comment]
安装Vimium扩展可以使用Vim的方式操作浏览器

**Reply by tw93** [reply]
哈哈哈，说实话，我强迫自己用过 vim，还是没有达到这个境界，不过现在我几乎都是快捷键了

**Comment by CFNull** [comment]
很好，适合懒人。全抄过去了。

**Reply by tw93** [reply]
哈哈哈，欢迎发出你的效果来看看

**Comment by czyt** [comment]
用vivaldi 书签和去广告的都可以不装了

**Reply by tw93** [reply]
这个有一说一 稍微有些复杂了

**Comment by kneep** [comment]
在macOS上，如果不用原生的UI，流畅度比较差。

**Reply by tw93** [reply]
原生ui主要是指 Safari 吗？

**Reply by kneep** [reply]
是的

**Comment by elizen** [comment]
太好用了！赶紧设置了一遍。

**Reply by tw93** [reply]
哈哈哈 喜欢就好

**Comment by Sunsh4j** [comment]
记得博主推过一个从 URL 中移除跟踪元素的扩展，怎么没写上，不用了吗？

**Reply by tw93** [reply]
那个挺好用，不过由于我平时开发，用浏览器调试很多，有时候一些埋点参数干掉了，不是很方面，不是前端还是比较推荐实用的

**Comment by CY-Christin** [comment]
竟然没有人用 opera，opera 新版我个人觉得非常美观好用，至于垂直侧边 tab 我记得有单独的 chrome 插件支持，但是 opera 的侧边栏有别的用处，我就没有用。

**Comment by umiii37** [comment]
用小蓝鸟替换 Twitter 的 X Logo←我真的笑出声，你太可爱了哈哈哈哈

**Comment by dengweiqiang** [comment]
你好，Maple NewTab这个插件，有两个地方感觉可以改进一下。

1. 标签页的高清图，似乎无法刷新出新图片，我看了一下控制台，不知道是不是bing没有返回的问题。
![image.png](https://dengwq.oss-cn-hangzhou.aliyuncs.com/img/20230904142535.png)

2. 右上角的选择框，没有办法触发，有时重启浏览器后，能够看到，但是选择框的文字和背景色重叠，使得只有鼠标移动到文字上，才能看清（没有截到合适的图片）。
![image.pnghttps://dengwq.oss-cn-hangzhou.aliyuncs.com/img/20230904142303.png)]

**Comment by eluotao** [comment]
bookmarks 能支持自定义长度就好了，感觉长度有点不够。主要是鼠标滑动不舒服，不太容易滑倒想要的地方，反正是快捷键用完就隐藏，想直接到浏览器底部去。

**Reply by eluotao** [reply]
自定义 长度 高度

**Comment by Beeta** [comment]
你好，之前收藏了你一篇关于alacrity的文章：https://tw93.fun/2023-01-25/alacritty.html 现在怎么找不到了？

**Reply by tw93** [reply]
https://tw93.fun/2023-02-06/alacritty.html
是不是这个

**Comment by duofu** [comment]
> ‘Maple Bookmarks’ 老实说有点慢（书签虽然整理过，但我的书签比较多），性能上不如 ‘Omni - Bookmark, History, & Tab Manager’

> ‘Maple NewTab’ 与 ‘TabX’ 一样，简洁首页，[doge] 对很多万年不重启、开N个Chrome窗口的人来说，还是‘SD2’来的更实用些

> ‘uBlock Origin’、‘Adblock plus’、‘Adguard’，更倾向于‘AdGuard’，体积小（但它阻止不了很多的弹窗）

> ‘Tampermonkey’油猴、‘Violentmonkey’暴力猴，没有很多定制化脚本需求的话，Chrome或者Edge自带的扩展管理程序就够用了

**Reply by dengweiqiang** [reply]
sd2好用。
omni好像不能在新建标签页上使用，比较鸡肋。
Maple Bookmarks 用了一段时间发现还不如ctrl + shift + o 来的方便。
Maple NewTab 不能更新，用了一段时间，删了。

**Comment by Sunsh4j** [comment]
uBlock Origin确实是很轻便，但是现在观看YouTube的视频的时候，会出现随机弹窗「YouTube 不允许使用广告拦截器」，这个有办法解决下嘛🤣🤣

**Reply by tw93** [reply]
没有碰到过，是不是你的 uBlock 拦截的东西开得太多了，建议就开几个就好了

**Reply by Sunsh4j** [reply]
是YouTube升级了广告策略🤣🤣
另外，想请教下Pake项目Releases里的安装包是有专门针对网站做什么优化吗，还是直接默认编译打包的呢？
我记得之前反馈过，安装后，软件的托盘图标全都是默认的🤣🤣

**Reply by tw93** [reply]
这里应该是Bug，我有空来修复一下，奈何我平时用mac...

**Reply by Sunsh4j** [reply]
也不是大问题，不影响使用，就是看着别扭🤣

**Comment by BonjourYY** [comment]
请问 Edge 浏览器的个人账户头像怎么移动到左边？

**Comment by chengnan049** [comment]
有错别字，清楚浏览器纪录 > 清除浏览记录

**Comment by justincnn** [comment]
目前看到的最棒的关于浏览器的优化文章，我搬运到自己的博客做备份啦，
ps：会备注大神的来源，感谢

**Reply by tw93** [reply]
其实我不建议搬运博客到自己博客备份，因为这个文章在我github上不会删除的

**Comment by jwjw1100** [comment]
https://typefully.com/Minimal Theme for Twitter我用了现在打不开twitter

**Reply by jwjw1100** [reply]
ERR_CONNECTION_RESET
